tests/unit/test_charm.py

```diff
@@ -62,7 +62,8 @@ class MockStat():
         harness.charm._stored.config = default_config
         action_event = Mock()
         harness.charm._on_update_status(action_event)
-        assert harness.model.unit.status == BlockedStatus('Last sync: Thu Jan  1 01:00:01 1970 '
+        print(harness.model.unit.status)
```

Comment:
Did you remember to remove it?

Reply:
Yes, this was used only for my local tests.